scanning/disa-alignment/ansible.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+import os
+import subprocess
+
+import shared
+from lib import util, results, virt, versions, ansible
+from conf import partitions, remediation
+
+
+ansible.install_deps()
+virt.Host.setup()
+
+g = virt.Guest('minimal_with_oscap')
+
+if not g.can_be_snapshotted():
+    ks = virt.Kickstart(partitions=partitions.partitions)
+    g.install(kickstart=ks)
+    g.prepare_for_snapshot()
+
+# the VM guest ssh code doesn't use $HOME/.known_hosts, so Ansible blocks
+# on trying to accept its ssh key - tell it to ignore this
+os.environ['ANSIBLE_HOST_KEY_CHECKING'] = 'False'
+
+with g.snapshotted():
+    playbook = util.get_playbook(shared.profile)
+    skip_tags = ','.join(remediation.excludes())
+    skip_tags_arg = ['--skip-tags', skip_tags] if skip_tags else []
+    ansible_cmd = [
+        'ansible-playbook', '-v', '-i', f'{g.ipaddr},',
+        '--private-key', g.ssh_keyfile_path,
+        *skip_tags_arg,
+        playbook,
+    ]
+    util.subprocess_run(ansible_cmd, check=True)
+    g.soft_reboot()
+
+    with util.get_content() as content_dir:
+        # There is always one (the latest) DISA benchmark in content src
+        references = content_dir / 'shared' / 'references'
+        disa_ds = next(
+            references.glob(f'disa-stig-rhel{versions.rhel.major}-*-xccdf-scap.xml')
+        )
+
+        content_report, content_arf = shared.content_scan(g, util.get_datastream())
+        disa_report, disa_arf = shared.disa_scan(g, disa_ds)
+
+        # Compare ARFs via CaC/content script and report results from output
+        compare_script = content_dir / 'utils' / 'compare_results.py'
+        env = os.environ.copy()
+        env['PYTHONPATH'] = str(content_dir)
+        cmd = [
+            compare_script, content_arf, disa_arf,
+        ]
```

Comment:
If you're going to the effort of defining `cmd` separately using a multi-line syntax, I would embed `compare_script` into it:
```python
cmd = [
    content_dir / 'utils' / 'compare_results.py',
    content_arf,
    disa_arf,
]
```
alternatively,
```python
cmd = [compare_script, content_arf, disa_arf]
```